srai/models/gtfs2vec_model.py

```diff
@@ -0,0 +1,72 @@
+"""
+Embedding model for gtfs2vec.
+
+This module contains embedding model from gtfs2vec paper [1].
+
+References:
+    [1] https://doi.org/10.1145/3486640.3491392
+"""
+from typing import Any
+
+import torch
+from pytorch_lightning import LightningModule
+from torch import nn
+from torch.nn import functional as F
+
+
+class GTFS2VecModel(LightningModule):  # type: ignore
+    """Autoencoder based embedding model for gtfs2vec."""
+
+    def __init__(
+        self,
+        n_features: int,
+        n_hidden: int = 48,
+        emb_size: int = 64,
+    ) -> None:
+        """
+        Init GTFS2VecModel.
+
+        Args:
+            n_features (int): Number of features.
+            n_hidden (int, optional): Number of hidden units. Defaults to 48.
+            emb_size (int, optional): Embedding size. Defaults to 64.
```

Comment:
Since you're using the n_something format, maybe it would make sense for this to be `n_embed` or something? I suppose this might need a naming convention later on to make it consistent across the codebase